### PR TITLE
[Endpoint][Response Actions] API docs for Action Details and Action List

### PR DESCRIPTION
**Pull request by paul-tavares**
# Description

Add the following documentation pages:

- Action Details API ([preview](https://security-docs_2269.docs-preview.app.elstc.co/guide/en/security/master/get-action-api.html))
- Action List API  ([preview](https://security-docs_2269.docs-preview.app.elstc.co/guide/en/security/master/list-actions-api.html))

### CODE REVIEW

**Comment by paul-tavares** [comment]
@elastic/security-docs  - FYI. Looking for reviews when you get a chance

**Review by kevinlog — APPROVED** [review]

**Review by joeypoon — APPROVED** [review]

File: docs/management/api/list-actions-api.asciidoc [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+* `isolate`
+* `unisolate`
+* `kill-process`
+* `suspend-process`
+* `running-processes`
+
+|
+|`agentIds` |string[] |A list of agent ids. Max of 50. |
+|`userIds` |string[] |A list of user ids. |
```
Comment (jmikell821) [inline docs/management/api/list-actions-api.asciidoc:33]:
```suggestion
|`userIds` |string[] |A list of user IDs. |
```

File: docs/management/api/list-actions-api.asciidoc [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+List actions:
+
+[source,sh]
+--------------------------------------------------
+GET /api/endpoint/action
+--------------------------------------------------
+// KIBANA
+
+List actions in the last 24 hours for two specific agent ids and commands of either `isolate` or `kill-process`:
```
Comment (jmikell821) [inline docs/management/api/list-actions-api.asciidoc:51]: Want to confirm what this code is saying. Is it listing all `isolate` or `kill-process` commands in the past 24 hours for two specific agent IDs? 

If so, my recommendation: 
```suggestion
Lists `isolate` or `kill-process` command actions in the last 24 hours for two specific agent IDs:
```

Sorry if I misunderstood!

Comment (paul-tavares) [reply docs/management/api/list-actions-api.asciidoc:51]: @jmikell821 - Yes, exactly right. Thanks for the re-wording 😄

**Comment by paul-tavares** [comment]
@jmikell821 I applied your changes. Can you take another look when you can?

🙏

**Review by benironside — APPROVED** [review]
LGTM

File: docs/management/api/list-actions-api.asciidoc [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+* `unisolate`
+* `kill-process`
+* `suspend-process`
+* `running-processes`
+
+|
+|`agentIds` |string[] |A list of agent IDs. Max of 50. |
+|`userIds` |string[] |A list of user IDs. |
+|`startDate` |string |A start date in ISO format or {ref}/common-options.html#date-math[Date Math format]. |
+|`endDate` |string |A start date in ISO format or {ref}/common-options.html#date-math[Date Math format]. |
```
Comment (joepeeples) [inline docs/management/api/list-actions-api.asciidoc:35]:
```suggestion
|`endDate` |string |A end date in ISO format or {ref}/common-options.html#date-math[Date Math format]. |
```

**Review by joepeeples — COMMENTED** [review]
One small correction, otherwise looks good! Thanks!

**Review by joepeeples — APPROVED** [review]
LGTM!

**Pull request merged**